src/client/core/aad/auth-provider.ts

```diff
@@ -18,11 +20,17 @@ export type AuthorizationResult = AuthenticationResult;
 export default class AuthProvider {
     private _clients: StringMap<ClientApplication> = {};
     private _accounts: StringMap<AccountInfo> = {};
+    private _cachePlugin: MSALCachePlugin;
+    private _logoutPromise: Promise<void>;
```

Comment:
`_logoutPromise` and `_primaryClient` should be marked as optional since `undefined` is a valid value for each of them